config/cloud.cfg.tmpl

```diff
@@ -125,8 +125,8 @@ cloud_config_modules:
 {% if variant in ["rhel"] %}
  - rh_subscription
 {% endif %}
-{% if variant in ["rhel", "fedora", "photon"] %}
-{% if variant not in ["photon"] %}
+{% if variant in ["rhel", "fedora", "openmandriva", "photon"] %}
+{% if variant not in ["openmandriva", "photon"] %}
```

Comment:
It is possible for an OpenMandriva system to be managed by a Spacewalk/Uyuni deployment, since there's a DNF plugin for it.

Reply:
True. I was careful about it because it's untested, but yes, it's likely to work

Reply:
Yes, I was overly cautious because it's untested - but should work. Fixed.